attestation-service/verifier/src/tdx/mod.rs

```diff
@@ -10,9 +10,9 @@ use eventlog::{CcEventLog, Rtmr};
 use quote::{ecdsa_quote_verification, parse_tdx_quote};
 use serde::{Deserialize, Serialize};
 
-mod claims;
+pub(crate) mod claims;
```

Comment:
Just a nit: do we need this?

Reply:
I think so, since we reference that (sub-)module from the az-tdx-vtpm sibling module of the tdx module (to share code)

```rust
use super::tdx::claims::generate_parsed_claim;
```